docs/docs/classic-ml/webhooks/index.mdx

```diff
@@ -0,0 +1,413 @@
+import { APILink } from "@site/src/components/APILink";
+
+# Webhooks
+
+## Overview
+
+MLflow webhooks enable real-time notifications when specific events occur in the Model Registry. When you register a model, create a new version, or modify tags and aliases, MLflow can automatically send HTTP POST requests to your specified endpoints. This enables seamless integration with CI/CD pipelines, notification systems, and other external services.
+
+## Key Features
+
+- **Real-time notifications** for Model Registry events
+- **HMAC signature verification** for secure webhook delivery
+- **Multiple event types** including model creation, versioning, and tagging
+- **Built-in testing** to verify webhook connectivity
+
+## Supported Events
+
+MLflow webhooks support the following Model Registry events:
+
+<table>
+  <thead>
+    <tr>
+      <th>Event</th>
+      <th>Description</th>
+      <th>Example Payload</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>REGISTERED_MODEL_CREATED</code></td>
+      <td>Triggered when a new registered model is created</td>
+      <td>
+
+        ```json
+        {
```

Comment:
q: do we think example payload is enough for the documentation? Or should we actually document the concrete schema with optional/required?

Reply:
Actually, let's link to specific payload class under `mlflow.webhooks.types`

Reply:
Sounds good. I'll add a page for `mlflow.webhooks.types` and link it here.